### PR TITLE
feature/admin sees list of bought rewards by employee

### PR DESCRIPTION
**Pull request by kajawitek**
Hi Nerds! 🤓

I added new features to our app:

  1. Admin has list of bought rewards by employee view.
  2. Specs check that admin sees employee's bought rewards list.

### CODE REVIEW

File: app/views/admins/employees/show.html.erb [hunk 1]
```diff
@@ -1,9 +1,38 @@
 <p id="notice"><%= notice %></p>
 
 <p>
-  <strong>Title:</strong>
-  <%= @employee.email %>
+  <h1>Employee information</h1>
+  <strong>Email:</strong>
+  <%= @employee.email %></br>
```
Comment (ciejj) [inline app/views/admins/employees/show.html.erb:6]: missing indentation

File: app/views/admins/employees/show.html.erb [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
+  <strong>Points:</strong>
+  <%= @employee.number_of_available_points %></br>
+  <strong>Available kudos:</strong>
+  <%= @employee.number_of_available_kudos %>
 </p>
-
 <%= link_to 'Edit', edit_admins_employee_path(@employee) %> |
 <%= link_to 'Back', admins_employees_path %>
+<p>
+  <h1>Employee orders</h1>
```
Comment (ciejj) [inline app/views/admins/employees/show.html.erb:15]: consider putting it in a partial

**Review by ciejj — APPROVED** [review]
Nice one!

File: app/controllers/admins/employees_controller.rb [hunk 3]
```diff
@@ -6,6 +6,7 @@ def index
 
     def show
       @employee = Employee.find(params[:id])
+      @orders = @employee.orders
```
[the remark this answers is not in the document]
Comment (kajawitek) [reply app/controllers/admins/employees_controller.rb]: done :)

File: spec/factories/orders.rb [hunk 4]
```diff
@@ -2,5 +2,6 @@
   factory :order do
     reward
     employee
+    purchase_price { 10 }
```
[the remark this answers is not in the document]
Comment (kajawitek) [reply spec/factories/orders.rb]: good idea - done :)

File: app/views/admins/employees/show.html.erb (hunk 1 above)
Comment (kajawitek) [reply app/views/admins/employees/show.html.erb]: added :)

File: app/views/admins/employees/show.html.erb (hunk 2 above)
Comment (kajawitek) [reply app/views/admins/employees/show.html.erb]: done :)

**Pull request merged**